### PR TITLE
[bug]Resolve the problem that the flink jobid stored in the trackid i…

### PR DESCRIPTION
**Pull request by MonsterChenzhuo**
## What changes were proposed in this pull request

Issue Number: Involving #1423 

## Brief change log

For the precise determination of the state logic, we are using the archive log of jobmanage, but the query of the archive log has become a problem, because the previous logic, trackid the object, the value of jobid is null, resulting in you can not go to the corresponding query, so propose the current pr

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): no

### CODE REVIEW

**Comment by MonsterChenzhuo** [comment]
hi @wolfboys ，I have made the update and can continue with the code review

**Comment by wolfboys** [comment]
> hi @wolfboys ，I have made the update and can continue with the code review

thanks for your contribution, I'll review it later

File: streampark-common/src/main/scala/org/apache/streampark/common/conf/ConfigConst.scala [hunk 1]
```diff
@@ -48,6 +48,8 @@ object ConfigConst {
 
   val KEY_JOB_ID = "jobId"
 
+  val KEY_FLINK_JOB_ID = "flinkJobId"
```
Comment (wolfboys) [inline streampark-common/src/main/scala/org/apache/streampark/common/conf/ConfigConst.scala]: hi MonsterChenzhuo:
I recommend to add `id` and `jobId` field in `SubmitRequest`, Don't add it to the global constant

**Comment by MonsterChenzhuo** [comment]
@wolfboys Thanks for the huge review work, I have revised all your suggestions.

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**